app/views/questions/show.html.erb

```diff
@@ -0,0 +1,56 @@
+<p id="notice"><%= notice %></p>
+  <div class="question_info">
+    <div class="question_title">
+      <%= @question.title %>
+    </div>
+    <div class="seperator">
+    </div>
+    <div class="question_detail">
+      <%= @question.detail %>
+    </div>
+    <div class="time">
+      <br>
+      <% if can? :update, @question %>
+        <%= link_to edit_question_path(@question) do %> 
+          <span class="edit_delete"> edit </span>
+        <% end %>
+      <% end %>
+      <% if can? :destroy, @question %>
+        <%= link_to question_path(@question), method: :delete, data: { confirm: 'Are you sure?' } do %>
+          <span class="edit_delete"> delete </span>
```

Comment:
this span doesn't contain the link to edit then why the class name is `edit_delete`? Also only `delete` is not a good class name. it can be `delete-question` / `delete-answer`